packages/dialtone-css/lib/build/less/components/skeleton.less

```diff
@@ -17,15 +17,19 @@
 //  ============================================================================
 //  @   SKELETON CONTAINER
 //  ----------------------------------------------------------------------------
+
+// The --placeholder-from-color and --placeholder-to-color
+// custom properties can be set on the parent class of the
+// placeholder to control the animation colors.
 .skeleton-placeholder {
```

Comment:
Namespace with `d-` maybe? Would that be a breaking change to have to update for non-Vue use?

Reply:
Yeah, but we've already done a few those already 🤷.